package.json

```diff
@@ -10,6 +10,7 @@
     "format:ci": "pnpm -r run format:src --check",
     "format:md": "remark *.md -o --use @kadena-dev/markdown",
     "preinstall": "npx --no-install only-allow pnpm",
+    "knip": "NEXT_TRANSLATE_PATH=packages/apps/tools knip",
```

Comment:
```
NEXT_TRANSLATE_PATH=packages/apps/tools 
```

This has nothing to do with Knip, but some dependency uses `process.cwd()` instead of properly resolving the current Next.js directory and this is the only way I see now. Another could be to `pnpm patch` the package (and/or open PR upstream), will look into that later.